### PR TITLE
Azure datalake storage

### PR DESCRIPTION
**Pull request by dillonroach**
Checklist

- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ x] Source is from official source
- [ x] Package does not vend other packages
- [ x] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@scopatz @costrouc

### CODE REVIEW

File: recipes/azure-datalake-store/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 55bcbec99a35a52ea291518dadfd94c3b645e66d05ce927228289b93979ad3a5
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```
Comment (marcelotrevisani) [inline recipes/azure-datalake-store/meta.yaml]: Could you please change it to
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/azure-datalake-store/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    providing pythonic file-system and file objects, seamless transition
+    between Windows and POSIX remote paths, high-performance up- and down-loader.
+  doc_url: http://github.com/Azure/azure-data-lake-store-python
+  dev_url: http://github.com/Azure/azure-data-lake-store-python
+
+extra:
+  recipe-maintainers:
+     - dillonroach
+     - scopatz
+     - costrouc
```
Comment (marcelotrevisani) [inline recipes/azure-datalake-store/meta.yaml]: @scopatz and @costrouc 
Are you willing to be a maintainer of this recipe as well?

Comment (costrouc) [reply recipes/azure-datalake-store/meta.yaml]: Yes

**Comment by dillonroach** [comment]
Curious - the linter is mad after the update commit - 
`There are 1 too many lines. There should be one empty line at the end of the file.` but the recipe only has one extra line currently at EoF.

File: recipes/azure-datalake-store/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/azure-datalake-store/meta.yaml]: yep

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**